### ISSUE TITLE
CaaSP 3 Admin Guide fails to build

### ISSUE THREAD

**Issue by ghost** [openSUSE/daps2docker#8]
```
daps2docker DC-caasp-admin pdf

invalid symlink "/tmp/daps2docker-n4k3KyJ3/.profiled/noprofile/autoyast_example_adminnode.xml" -> "../../../xml/autoyast_example_adminnode.xml"
```

**Comment by ghost** [comment]
@eroca has a fix already, I hear.

**Comment by ghost** [comment]
Fixed by 911c83c
